lib/connect-session-sequelize.js

```diff
@@ -25,7 +29,20 @@ module.exports = function SequelizeSessionInit(Store) {
 		}
 		Store.call(this, options);
 
+		var key;
+		this.options = {};
+		for (key in defaultOptions) {
```

Comment:
I wasn't aware of util._extend and didn't want to add a library you didn't already have.